compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/FirDefaultErrorMessages.kt

```diff
@@ -444,6 +446,8 @@ class FirDefaultErrorMessages : DefaultErrorMessages.Extension {
                 "An anonymous function is not allowed to specify default values for its parameters"
             )
             map.put(USELESS_VARARG_ON_PARAMETER, "Vararg on this parameter is useless")
+            map.put(MULTIPLE_VARARG_PARAMETERS, "Multiple vararg-parameters are prohibited")
+            map.put(FORBIDDEN_VARARG_PARAMETER_TYPE, "\"Forbidden vararg parameter type: {0}\", RENDER_TYPE", RENDER_TYPE)
```

Comment:
The message looks weird: why does it include "RENDER_TYPE" and quotation marks?

Reply:
Yep that's exactly a copy paste error :(

AFAIK, there is no test covering the actual error message.

Reply:
Maybe we can use this? https://github.com/JetBrains/kotlin/blob/master/compiler/testData/cli/jvm/firError.out